backend/src/interface/persistent_data/model.rs

```diff
@@ -31,6 +34,7 @@ pub struct Line {
 }
 
 /// Struct for database-operations. Related to the database entity 'meal'.
+#[derive(Debug, PartialEq)]
```

Comment:
add Eq aswell

Reply:
That won't work, because meal contains an f32, which cannot be Eq'ed